tutorials/measurement_chain.ipynb

```diff
@@ -0,0 +1,553 @@
+{
```

Comment:
<p>double article: we add a an equip.</p><p> </p><p>Here you say, that we need to provide the name of the source or transformation, but earlier on get_transformation, you didn't comment on the argument. I recommend doing the same thing there.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>